### PR TITLE
feat: Add support for VS 2019 16.6 debugger

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -135,7 +135,23 @@ For the time being, you will also need to make sure that mscorlib is disabled in
...
+To enable the debugging, add the following line to your `launchSettings.json` file:
+```
+"inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}"
+```
+
+in every profile section of the file, below each `"launchBrowser": true,` line.
+
+Press `F5` to start debugging.
+
+Note that that breakpoints in the main entry point of the executable are currently ignored.
```
Comment (davidjohnoliver) [inline Readme.md]:
```suggestion
Note that breakpoints in the main entry point of the executable are currently ignored.
```

File: src/Uno.Wasm.Bootstrap/Constants.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@ namespace Uno.Wasm.Bootstrap
 	internal class Constants
 	{
 		// NOTE: The SDK version may be overriden by an installation of the https://www.nuget.org/packages/Uno.Wasm.MonoRuntime nuget package
-		public const string DefaultSdkUrl = @"https://unowasmbootstrap.blob.core.windows.net/runtime/mono-wasm-b777471fcac.zip";
-		public const string DefaultAotSDKUrl = "https://unowasmbootstrap.blob.core.windows.net/runtime/wasm-release-Linux-b777471fcace85325e2c2af0e460f4ecd8059b5a.zip";
+		public const string DefaultSdkUrl = @"https://jenkins.mono-project.com/job/test-mono-mainline-wasm/label=ubuntu-1804-amd64/4892/Azure/processDownloadRequest/4892/ubuntu-1804-amd64/sdks/wasm/mono-wasm-1add5cbc06a.zip";
```
Comment (davidjohnoliver) [inline src/Uno.Wasm.Bootstrap/Constants.cs]: What does this change imply?

Comment (jeromelaban) [reply src/Uno.Wasm.Bootstrap/Constants.cs]: Nothing, I'll revert it to use our own storage, I'm just validating that the test are passing :)

**Pull request merged**